tools/public_api_guard/cdk/table.d.ts

```diff
@@ -244,6 +245,13 @@ export interface CellDef {
     template: TemplateRef<any>;
 }
 
+export declare class CoalescedStyleScheduler {
```

Comment:
Can we exclude this from the public-api?

Reply:
I'm setting it to docs-private, but I will need to be able to import it from experimental for my next PR.

Reply:
Ack- Can you prefix the class name with an underscore for now (surpressing the lint)? We just decided this morning to get more aggressive about prefixing things like this to make the private-ness more clear